.github/workflows/test_agents.yaml

```diff
@@ -0,0 +1,34 @@
+on:
+  pull_request: {}
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        include:
+          - extra: ""
+            script: "test_basic.py"
+            python-version: "3.10"
+          - extra: "stable-baselines"
+            script: "test_sb.py"
+            python-version: "3.7"
+          - extra: "stable-baselines3"
+            script: "test_sb3.py"
+            python-version: "3.10"
+          - extra: "ray-rllib"
+            script: "test_ray_rllib.py"
+            python-version: "3.10"
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        with:
+          python-version: ${{matrix.python-version}}
+      - run: python3 -m pip install wheel==0.38.4
+      - run: python3 -m pip install .[tests]
+      - if: ${{ matrix.extra != '' }}
+        run: python3 -m pip install .[${{matrix.extra}}]
+      - if: ${{ matrix.extra == 'stable-baselines' }}
+        run: python3 -m pip install tensorflow==1.14.0
+      - run: git clone https://github.com/diambra/agents.git agents
```

Comment:
Use actions/checkout@v3 here instead. You can also specify other repos with that.

Reply:
Thanks!